test/app/db/schema.rb

```diff
@@ -29,6 +29,7 @@
   create_table "users", force: :cascade do |t|
     t.string "name", null: false
     t.integer "age", null: false
+    t.integer "status"
     t.string "phone_numbers"
     t.string "contact_info"
     t.string "family_tree"
```

Comment:
I found these columns are not defined in the migration files. We need to add them also.